python/lsst/ip/isr/calibType.py

```diff
@@ -262,8 +263,8 @@ def readFits(cls, filename):
         try:
             with warnings.catch_warnings("error"):
                 newTable = Table.read(filename, hdu=extNum)
-            tableList.append(newTable)
-            extNum += 1
+                tableList.append(newTable)
+                extNum += 1
```

Comment:
Does this actually change any behavior?

Reply:
The `with warnings.catch_warnings("error")` is needed because astropy will read the "first available" table if the requested table does not exist, throwing a warning instead of an exception.  This promotes the warning to an exception, and ensures that the list represents what's in the FITS table.

Reply:
Wow, that is subtle. Sounds good!